tools/python/requirements.txt

```diff
@@ -4,5 +4,5 @@ pandas
 tabulate
 requests
 web3==5.24.0
```

Comment:
looks like this needs to be unpinned, due to it being dictated by `bal_addresses`

Reply:
fixed in 3adba26